### PR TITLE
Add continous integration action

### PR DESCRIPTION
**Pull request by omorsi**
This PR adds a very basic continuous integration action that makes sure that all go files build successfully and that all tests pass successfully.

### CODE REVIEW

**Comment by omorsi** [comment]
Not sure why the action didn't run on pull request as it did on my fork: https://github.com/omorsi/scep/pull/6.

@pavolmarko FYI!

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
```
Comment (groob) [inline .github/workflows/CI.yml:19]: Is 1.16 not available yet?

Comment (groob) [reply .github/workflows/CI.yml:19]: How about this? 

```
jobs:
  test-build:
    name: Test & Build
    strategy:
      matrix:
        go-version: [1.15.x, 1.16.x]
        platform: [ubuntu-latest, macos-latest, windows-latest]
    runs-on: ${{ matrix.platform }}
    steps:
    - name: setup go
      uses: actions/setup-go@v2
      with:
        go-version: ${{ matrix.go-version }}
      id: go

```

**Review by groob — APPROVED** [review]

File: .github/workflows/CI.yml (hunk 1 above)
Comment (omorsi) [reply .github/workflows/CI.yml]: Added 1.16.
Thanks for letting me know about matrices in github actions, I have added them.
Do we need the `id: go` part?

Comment (groob) [reply .github/workflows/CI.yml]: not sure, I copied it from somewhere else :)

**Pull request merged**